Main.java

```diff
@@ -0,0 +1,50 @@
+package com.company;
+
+import java.util.Scanner;
+
+public class Main {
+    static String scanLine() {
+        Scanner scan = new Scanner(System.in);
+        return scan.nextLine();
+    }
+    public static void main(String[] args) {
+        int start, end, length, var, aux;
+        int mean = 0;
+        System.out.println("Arithmetic mean algorithm");
+        System.out.println("""
+            Write 1 or 2 depending the case:\s
+            1 - arithmetic mean of a certain number range\s
+            2 - arithmetic mean of a certain count of numbers""");
+        var = Integer.parseInt(scanLine());
+        switch (var) {

```

Comment:
La finalul fiecarui case, se pune break, ca sa inceteze executia urmatoarelor cazuri. Exista situatii unde var ar putea fi si 1 si 2 (daca de ex, cineva ar schimba var din 1 in 2 in case 1).